go/inst/instance_dao.go

```diff
@@ -3068,6 +3155,79 @@ func FigureInstanceKey(instanceKey *InstanceKey, thisInstanceKey *InstanceKey) (
 	return figuredKey, nil
 }
 
+// Obtain information about Group Replication  for this host as well as other hosts who are members of the same group
+// (if any). This method swallows up any errors, interpreting them as if the host does not support group replication
+// (e.g. because the group replication plugin is not loaded, therefore the corresponding performance schema tables and
+// global variables are not known to the server.
+func PopulateGroupReplicationInformation(instance *Instance, db *sql.DB) {
```

Comment:
I'm not sure ignoring the error is the best way here, I catch error and ignore only for expected errors if there is no group replication plugin, etc, but not ignore all errors

Reply:
Agree

Reply:
Is it not possible to validate whether this host at all supports group replication, and based on that finding to invoke/skip this function?

Reply:
Done.